src/serenity/utils.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef SERENITY_UTILS_HPP
+#define SERENITY_UTILS_HPP
+
+#include <vector>
+
+template <typename T>
+T flattenListsInsideVector(std::vector<T> _vectorOfContainers) {
+  T products;
```

Comment:
I would rather go https://github.com/AlbertoMarnetto/multidim. Also I dislike the fact, that this code is in ambigous 'utils' - so I wondered about 'flattenedConsumables' method in Consumer class with this library #154